extensions/.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+	extends: [ 'plugin:jest/recommended' ],
+	plugins: [ 'jsx-a11y', 'jest' ],
+	env: {
+		'jest/globals': false,
+	},
+	overrides: [
+		{
+			files: [ '**/test/*.js' ],
+			env: {
+				'jest/globals': true,
+			},
+		},
+	],
+	rules: {
```

Comment:
Are these rules here again (in addition to overrides [in root](https://github.com/Automattic/jetpack/pull/11819/files#diff-e4403a877d80de653400d88d85e4801aL104)) because we can't cover `'extensions/**/test/*.js'` in both configs simultaneously?

Reply:
Should we rather bring overrides from root to here, not to need define rules twice here?

Reply:
They shouldn't be duplicated and this PR really isn't ready for much scrutiny. I'm tempted to dump it and start from scratch, possibly preparing an updated eslint package from Calypso first.